addon/components/md-btn.js

```diff
@@ -12,12 +12,18 @@ export default Component.extend(UsesSettings, {
   wavesClass: 'waves-light',
   text: null,
   icon: null,
-  iconPosition: oneWay('_mdSettings.buttonIconPosition'),
+  iconPosition: null,
   buttonType: null,
   actionArg: null,
   isFlat: computed.equal('buttonType', 'flat'),
   isDisabled: false,
 
+  didReceiveAttrs() {
```

Comment:
it would make more sense to put this on the init hook. `didReceiveAttrs` is called every time the attrs on a component change (i.e., updating values through a binding)